starter_code/script.js

```diff
@@ -3,18 +3,48 @@ var aiPoint = 0;
 
 // This function returns the selection of the computer
 function getAISelection() {
-    //TODO: randomly choose between 'rock', 'paper', or 'scissors'
+    var random = Math.random();
+    if(random < 1/3){
+        return 'rock';
+    }
+    
+    if (random < 2/3){
+        return 'scissor';
+    }
+    
+    return  'paper';
+    
 }
 
 // This function picks the winner
 function pickWinner(userValue, aiValue) {
     //TODO: pick the correct winner: user or ai
     //TODO: Add one point for the winner
+    if(userValue === aiValue){
+        return 'draw';
+    }
+    
+    if(userValue === 'rock' && aiValue === 'paper' ||
+       userValue === 'paper' && aiValue === 'scissor' ||
+       userValue === 'scissor' && aiValue === 'rock'){
+           aiPoint++;
+           return 'AI WINS!!!'
+       }
+    
+    if(userValue === 'paper' && aiValue === 'rock' ||
+       userValue === 'rock' && aiValue === 'scissor' ||
+       userValue === 'scissor' && aiValue === 'paper'){
+        userPoint++;
+        return 'user';
+    }
```

Comment:
hmm, if its not a tie, and the winner is not AI, then do we have to check again before just returning `user`?